src/dynamic_graph/sot/talos/talos.py

```diff
@@ -65,14 +65,18 @@ def __init__(self, name, device=None, tracer=None, fromRosParam=False):
         }
 
         if fromRosParam:
-            print("Using ROS parameter \"/robot_description\"")
-            rosParamName = "/robot_description"
-            import rospy
-            if rosParamName not in rospy.get_param_names():
-                raise RuntimeError('"' + rosParamName + '" is not a ROS parameter.')
-            s = rospy.get_param(rosParamName)
-
-            self.loadModelFromString(s, rootJointType=pinocchio.JointModelFreeFlyer, removeMimicJoints=True)
+            ltimeStep=0.005
```

Comment:
Might be a bit easier to read as `ltimeStep = device.getTimeStep() if device is not None else 0.005`